index.js

```diff
@@ -5,14 +5,42 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "Magenta"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+    let profileDiv = document.createElement('div')
+    let img = document.createElement('img')
+    let name = document.createElement('h2')
+    let nickname = document.createElement('em')
 
-/***** Deliverable 4 *****/
+    profileDiv.id = ("profile")
+    img.src = traveler.photo
+    img.alt = traveler.name
+    name.textContent = traveler.name 
+    nickname.textContent = traveler.nickname
+    
+    let profiles = document.querySelector("#profile")
+    profileDiv.append(img, name, nickname)
+    profiles.append(profileDiv)
```

Comment:
For this deliverable, the idea is to find the *existing* DOM elements that are already part of the HTML instead of generating new DOM elements. Here's a solution that doesn't involve creating new elements:

```js
const img = document.querySelector("#profile img");
img.src = traveler.photo;
img.alt = traveler.name;

const name = document.querySelector("#profile h2");
name.textContent = traveler.name;

const nickname = document.querySelector("#profile em");
nickname.textContent = traveler.nickname;
```